include/mamba/core/fetch.hpp

```diff
@@ -66,6 +66,21 @@ namespace mamba
             return m_ignore_failure;
         }
 
+        inline void set_active_logs(bool yes)
+        {
+            m_active_logs = yes;
+        }
+
+        inline bool active_logs() const
+        {
+            return m_active_logs;
+        }
+
+        inline curl_off_t get_downloaded_size() const
+        {
+            return downloaded_size;
```

Comment:
do we need this? downloaded size is public.

Reply:
right, I could use directly `downloaded_size`